tensorflow/lite/micro/kernels/transpose_conv.cc

```diff
@@ -105,6 +106,19 @@ TfLiteStatus CalculateOpData(TfLiteContext* context, TfLiteNode* node,
         &data->params.quantized_activation_max,
         data->per_channel_output_multiplier, data->per_channel_output_shift,
         output_channels));
+
+    // Some models have int16 biases which must be converted to int64 for
+    // inference.
+    if (input->type == kTfLiteInt16) {
+      TFLITE_DCHECK(filter->type == kTfLiteInt8);
+      TFLITE_DCHECK(output->type == kTfLiteInt16);
+      if (bias->type == kTfLiteInt16) {
+        TFLITE_DCHECK(
+            context->RequestScratchBufferInArena(
+                context, GetTensorShape(bias).FlatSize() * sizeof(std::int64_t),
+                &(data->bias_converted_buffer_index)) == kTfLiteOk);
```

Comment:
seems wasteful from the standpoint of memory footprint. Is it possibly better to have a flatbuffer modification so that int64 biases can be stored directly in the flatbuffer? Again, we don't need it fixed in this PR, but would be good to have a bug and circle back.

Reply:
Fortunately it seems to have little effect on overall memory, since it's a scratch buffer rather than a persistent buffer. Still, I filed b/192090531 and referenced it here to track this issue.